### PR TITLE
Add plugin package keyword recommendation

### PR DESCRIPTION
**Pull request by kkirsche**
Related to #15823, I would be interested in officially recommending a keyword such as `twbs-bootstrap` so that we can more easily showcase plugins and extensions for Bootstrap without requiring additional curation by the admins / collabs of TWBS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,6 +104,7 @@ Keep track of development and community news.
 - Read and subscribe to [The Official Bootstrap Blog](http://blog.getbootstrap.com).
 - Chat with fellow Bootstrappers in IRC. On the `irc.freenode.net` server, in the `##bootstrap` channel.
 - Implementation help may be found at Stack Overflow (tagged [`twitter-bootstrap-3`](http://stackoverflow.com/questions/tagged/twitter-bootstrap-3)).
+- Developers should use the keyword `twbs-bootstrap` on packages which modify or add to the functionality of Bootstrap when distributing through [NPM](https://www.npmjs.com/browse/keyword/twbs-bootstrap) or similar delivery mechanisms for maximum discoverability.
```
Comment (cvrebert) [inline README.md]: Please use lowercase `npm` for consistency.

**Comment by cvrebert** [comment]
@mdo SGTY?

**Comment by kkirsche** [comment]
Fixed capitalization and rebased

**Comment by mdo** [comment]
Why not just `bootstrap` as the keyword?

**Comment by kkirsche** [comment]
My reasoning was that `bootstrap` is used for many types of plugins already which are for actually bootstrapping an application which are not related to the framework. 

Secondly, it also allows for modification of it for version specific extensions which could help when V4 is released.

**Comment by mdo** [comment]
Most of the npm plugins I've found with a simple `bootstrap` search show something related to this project. I'd prefer us to go that route—I don't think we'd be running into much opposition and it keeps it a helluva lot simpler.

**Comment by kkirsche** [comment]
I'll change it if you'd prefer. Just for consistency with the tag on stackoverflow I figured having a single tag for anything and everything was simpler than having one tag for SO, one for plugins, etc.

**Comment by kkirsche** [comment]
Updated to `bootstrap` per your recommendation @mdo

**Comment by mdo** [comment]
:shipit:

**Pull request merged**